client/src/layouts/admin/component.tsx

```diff
@@ -1,35 +1,48 @@
 import ApplicationLayout from 'layouts/application';
-import Tabs from 'components/tabs';
+import Link from 'next/link';
+import classNames from 'classnames';
+
 import PageLoading from 'containers/page-loading';
 
-import { AdminLayoutProps } from './types';
 import { ADMIN_TABS } from './constants';
 
+import type { AdminLayoutProps } from './types';
+
 const AdminLayout: React.FC<AdminLayoutProps> = ({
   loading = false,
   currentTab,
-  headerButtons,
+  title = 'Admin',
   children,
 }: AdminLayoutProps) => (
   <ApplicationLayout>
     <section
       aria-labelledby="primary-heading"
-      className="min-w-0 flex-1 h-screen flex flex-col overflow-y-auto p-6 pb-16 lg:pb-0"
+      className="min-w-0 flex-1 h-screen flex flex-col overflow-y-auto bg-gray-100"
     >
       {loading && <PageLoading />}
 
-      <header className="sticky top-0 bg-gray-100 z-10 pt-6">
-        <h1 className="text-center md:text-left">Admin</h1>
+      <header className="sticky top-0 bg-primary z-20">
+        <div className="flex items-stretch justify-between px-12 rounded-tl-3xl bg-white border-b border-gray-200">
+          <h1 className="text-left my-8">{title}</h1>
 
-        <div className="flex flex-col md:flex-row justify-between items-center border-b border-gray-200">
-          <Tabs bottomBorder={false} activeTab={currentTab} tabs={ADMIN_TABS} />
-          {headerButtons && (
-            <div className="flex gap-3 mt-4 mb-4 md:-mt-2 md:mb-2">{headerButtons}</div>
-          )}
+          <div className="flex text-sm">
+            {Object.values(ADMIN_TABS).map((tab, index) => (
+              <Link key={tab.name} href={tab.href}>
+                <a
+                  className={classNames('flex items-center -mb-2px', {
+                    'ml-10': index !== 0,
```

Comment:
Could Tailwind's `first:` be used here? Not that it matters that much

Reply:
you are right